src/Sylius/Bundle/CoreBundle/Resources/config/validation/ProductReview.xml

```diff
@@ -16,5 +16,13 @@
         <constraint name="Sylius\Bundle\CoreBundle\Validator\Constraints\UniqueReviewerEmail" >
             <option name="groups">sylius_review</option>
         </constraint>
+        <property name="rating">
+            <constraint name="Range">
+                <option name="min">1</option>
+                <option name="max">5</option>
```

Comment:
I'm not sure if we should bound the upper limit, and at the very least the value should be configurable.

Reply:
The same constraint we have in `Sylius\Bundle\ApiBundle\Command\Catalog\AddProductReview`, and there's no option to pass parameters to a validation XML file. Therefore, we need to create a custom validator. However, I believe this is out of scope of this task and could we do in a separate PR.